Gemfile

```diff
@@ -3,10 +3,6 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in postgres_ext.gemspec
 gemspec
 unless ENV['CI']
-  if RUBY_PLATFORM =~ /java/
```

Comment:
@danmcclain I'm going to clean this PR up and merge it since it seems useful. Do you want this part in the final result?